xformers/ops/memory_efficient_attention.py

```diff
@@ -799,6 +826,7 @@ def from_arguments(
         value: torch.Tensor,
         attn_bias: Optional[Union[torch.Tensor, AttentionMask]] = None,
         p: float = 0.0,
+        has_custom_scale: bool = False,
```

Comment:
technically the argument to `memory_efficient_attention` is `scale` and not `has_custom_scale`. Let's try to keep the same arguments for `memory_efficient_attention` and `from_arguments`

Reply:
Make sense. Fixed.